marklogic-data-hub-central/ui/e2e/cypress/e2e/modeling/graphValidations.cy.tsx

```diff
@@ -266,84 +266,86 @@ describe("Graph Validations", () => {
     graphViewSidePanel.getPropertiesTab().click();
-    graphViewSidePanel.getPropertyName("ownedBy").should("be.visible");
+    graphViewSidePanel.getPropertyName("relTest1-Test2").should("be.visible");
 
-    cy.log("**Edist ownedBy property**");
-    propertyTable.editProperty("ownedBy");
+    cy.log("**Edist relTest1-Test2 property**");
+    propertyTable.editProperty("relTest1-Test2");
     propertyModal.clearPropertyName();
-    propertyModal.newPropertyName("owned-by");
-    cy.get(`[aria-label="customer-Id-option"]`).should("exist");
+    propertyModal.newPropertyName("rel-Test1-Test2");
+    cy.get(`[aria-label="test2-id-option"]`).should("exist");
     propertyModal.getSubmitButton().click();
     cy.wait(1000);
-    propertyTable.getProperty("owned-by").should("exist");
+    propertyTable.getProperty("rel-Test1-Test2").should("exist");
 
-    cy.log("**Checks that customer-Id cannot be deleted because is being used as foreign key**");
+    cy.log("**Checks that test2-id cannot be deleted because is being used as foreign key**");
     cy.get("#switch-view-table").click({force: true});
-    entityTypeTable.viewEntityInGraphView("Customer");
+    entityTypeTable.viewEntityInGraphView("a-Test2");
     graphViewSidePanel.getPropertiesTab().click();
-    propertyTable.getDeletePropertyIcon("Customer", "customer-Id").should("be.visible").click();
+    propertyTable.getDeletePropertyIcon("a-Test2", "test2-id").should("be.visible").click();
     confirmationModal.getDeletePropertyForeignKeyWarnText().should("be.visible");
     cy.findByText("Close").should("be.visible").click();
   });
 
   it("can view Entity's relationship from graph ", {defaultCommandTimeout: 120000}, () => {
-    modelPage.selectView("project-diagram");
-    graphVis.getPositionOfEdgeBetween("Customer,BabyRegistry").then((edgePosition: any) => {
+    cy.get("[data-cy=graph-view]").click({force: true});
+    graphVis.getPositionOfEdgeBetween("a-Test2,a-Test1").then((edgePosition: any) => {
       // Wait extended because of the delay of the animations
       cy.wait(150);
       cy.waitUntil(() => graphVis.getGraphVisCanvas().click(edgePosition.x, edgePosition.y, {force: true}));
     });
 
     relationshipModal.getModalHeader().should("be.visible");
-    relationshipModal.verifyRelationshipValue("owned-by");
-    relationshipModal.verifyForeignKeyValue("customer-Id");
+    relationshipModal.verifyRelationshipValue("rel-Test1-Test2");
+    relationshipModal.verifyForeignKeyValue("test2-id");
     relationshipModal.cancelModal();
   });
 
+
+  it("Deleting relation, entities and publish", () => {
+    cy.log("**Sort table by entities Name**");
+    cy.get("[data-testid='entityName']").scrollIntoView().should("be.visible").click();
+    entityTypeTable.getExpandEntityIcon("a-Test1");
+
+    cy.log("**Deletes relTest1-Test2 relationship**");
+    propertyModal.getDeleteIcon("a-Test1-relTest1-Test2").should("exist").scrollIntoView().should("be.visible").click();
```

Comment:
You renamed the property to rel-Test1-Test2
This gets stuck because it cna't find the old name.